src/osg/ContextData.cpp

```diff
@@ -163,7 +163,12 @@ unsigned int ContextData::createNewContextID()
 
             OSG_INFO<<"ContextData::createNewContextID() : reusing contextID="<<itr->first<<std::endl;
 
-            return itr->first;
```

Comment:
(comment about missing file not this file) The file CollectOcludersVisitor.cpp was changed for osg 3.4 but that change is not here.

Reply:
Nevermind, I see that it was fixed in osg proper.